docs/.vuepress/theme/components/LoE.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <article class="loe">
+    <header>
+      <h2 title="League of Entropy">
+        <img src="/images/logo-loe.png" alt="League of Entropy logo" />
+      </h2>
+      <p>
+        The <strong>League of Entropy</strong> is a collaborative project
+        between the founding members <strong>Cloudflare</strong>,
+        <strong>École polytechnique fédérale de Lausanne</strong>,
+        <strong>Kudelski Security</strong>, <strong>Protocol Labs</strong>, and
+        <strong>University of Chile</strong> to provide a verifiable,
+        decentralized randomness beacon. A decentralized randomness beacon
+        combines randomness from multiple independent high entropy sources to
+        generate a truly unbiased random number for anyone that may need a
+        public source of randomness.
+      </p>
+    </header>
+    <div class="history">
+      <h3>Latest Randomness</h3>
+      <p>
+        Here's the latest random value that was generated, round #<strong>{{
+          round
+        }}</strong
+        >:
```

Comment:
```suggestion
        }}</strong>:
```

Reply:
There is a formatter that does this. I hate it.